src/file.cpp

```diff
@@ -593,7 +593,7 @@ namespace libtorrent
 	{
 		ec.clear();
 		error_code e;
-		file_status s;
+		file_status s = {};
```

Comment:
what would you think about adding initializers to the members of this class instead?